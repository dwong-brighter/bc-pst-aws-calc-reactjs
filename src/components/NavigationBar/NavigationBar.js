import React from 'react';
import AppBar from '@material-ui/core/AppBar';
import IconButton from '@material-ui/core/IconButton';
import MenuButton from '@material-ui/icons/Menu';
import Toolbar from '@material-ui/core/Toolbar';
import Typography from '@material-ui/core/Typography';

function NavigationBar () {
  return (
    <AppBar position="static">
      <Toolbar variant="dense">
        <IconButton color="inherit" edge="start" aria-label="Menu">
          <MenuButton />
        </IconButton>
        <Typography variant="h6" color="inherit">BC PST Calculator for AWS Billing (Using ReactJS and Material-UI)</Typography>
      </Toolbar>
    </AppBar>
  );
}

NavigationBar.propTypes = {};

export default NavigationBar;
