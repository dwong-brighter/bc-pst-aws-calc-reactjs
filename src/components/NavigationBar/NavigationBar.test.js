import React from 'react';
import { shallow } from 'enzyme';
import NavigationBar from './NavigationBar';

describe('<NavigationBar />', () => {
  test('renders', () => {
    const wrapper = shallow(<NavigationBar />);
    expect(wrapper).toMatchSnapshot();
  });
});
