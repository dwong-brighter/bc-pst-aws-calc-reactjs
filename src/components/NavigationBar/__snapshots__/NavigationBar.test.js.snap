// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NavigationBar /> renders 1`] = `
<div>
  <WithStyles(ForwardRef(AppBar))
    position="static"
  >
    <WithStyles(ForwardRef(Toolbar))
      variant="dense"
    >
      <WithStyles(ForwardRef(IconButton))
        aria-label="Menu"
        color="inherit"
      >
        <MenuIcon />
      </WithStyles(ForwardRef(IconButton))>
      <WithStyles(ForwardRef(Typography))
        color="inherit"
        variant="h6"
      >
        BC PST Calculator for AWS Billing (Using ReactJS and Material-UI)
      </WithStyles(ForwardRef(Typography))>
    </WithStyles(ForwardRef(Toolbar))>
  </WithStyles(ForwardRef(AppBar))>
</div>
`;
